Modules/SubModule_ASL/xASL_wrp_Quantify.m

```diff
@@ -421,6 +422,15 @@ function xASL_wrp_Quantify(x, PWI_Path, OutputPath, M0Path, SliceGradientPath)
 
 xASL_io_SaveNifti(PWI_Path, OutputPath, CBF, 32, 0);
 
+if ~isempty(ATT)
```

Comment:
Perhaps nicer to have `pathOutput_CBF` and `pathOutput_ATT` from the start of this wrapper, this is a bit quick and dirty

Reply:
Agree done.